automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -154,11 +166,12 @@ def satellite6_capsule_zstream_upgrade():
     TO_VERSION
         Next satellite version to which satellite will be upgraded
     """
+    logger.highlight('\n========== CAPSULE UPGRADE =================\n')
     from_version = os.environ.get('FROM_VERSION')
     to_version = os.environ.get('TO_VERSION')
     if not from_version == to_version:
-        print ('Error! zStream Upgrade on Capsule cannot be performed as '
-               'FROM and TO versions are not same!')
+        logger.warning('zStream Upgrade on Capsule cannot be performed as '
```

Comment:
I believe to throw an error message, the error has to be in Product and shouldnt be in automation or configuration.